src/components/DayPicker.jsx

```diff
@@ -190,27 +240,33 @@ export default class DayPicker extends React.Component {
   }
 
   adjustDayPickerHeight() {
-    const $transitionContainer = $(ReactDOM.findDOMNode(this.refs.transitionContainer));
-    const heights = $transitionContainer.find('.CalendarMonth[data-visible="true"]').get().map(
-      (month) => this.getMonthHeight($(month))
-    );
+    const transitionContainer = ReactDOM.findDOMNode(this.refs.transitionContainer);
+    const calendarMonths = transitionContainer.querySelectorAll('.CalendarMonth');
+    const heights = [];
+
+    // convert node list to array
+    Array.prototype.slice.call(calendarMonths).forEach((el) => {
```

Comment:
spreading will compile down to effectively a slice + forEach - forEach.call will just be the forEach (so i'd recommend doing that)